docs/releases/6.0.txt

```diff
@@ -95,7 +95,9 @@ Minor features
 :mod:`django.contrib.sitemaps`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* The signed cookies session backend now supports custom session expiry
+  through the :meth:`.SessionBase.set_expiry` method. Previously the method
+  had no effect when using this backend.
```

Comment:
As this is a bug, I'm not sure we need a release note here. If we do, it should be under `django.contrib.sessions` and maybe a version changed note would be required (with updated docs) within "Using cookie-based sessions"